examples/tutorials/advanced/websockets-example-MNIST/start_websocket_servers.py

```diff
@@ -1,13 +1,12 @@
 import subprocess
-import sys
 
-python = "python" + sys.version[0:3]
 
-call_alice = [python, "run_websocket_server.py", "--port", "8777", "--id", "alice"]
+call_alice = ['python', "run_websocket_server.py", "--port", "8777", "--id", "alice"]
```

Comment:
Why changing this?

Reply:
It was giving error no such file exists ... so I tested what arguments in subprocess.Popen are doing by using these arguments ['notepad','README.md'] so it opened README.md so what that line what doing is opening that README.md file with notepad.exe so in windows there is only one python.exe file in each envirenment there is no python3.exe or python3.6.exe so in windows 10 if I activate some conda environment to execute this script it will use python.exe of that environment for that subprocess.Popen  ... I dont know if it will behave the same on linux I havn't tested it on that if you want I can make an if for when using windows and when using linux

Reply:
ok so I tried writing !python3 and !python2 and !python3.6 on google colab and its working there so it means these will work on Linux so I am adding an if statement for people using Linux